src/main/java/com/ryszardpanda/medicalClinic/controller/VisitController.java

```diff
@@ -0,0 +1,42 @@
+package com.ryszardpanda.medicalClinic.controller;
+
+import com.ryszardpanda.medicalClinic.mapper.VisitMapper;
+import com.ryszardpanda.medicalClinic.model.Visit;
+import com.ryszardpanda.medicalClinic.model.VisitDTO;
+import com.ryszardpanda.medicalClinic.service.VisitService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RestController
+@RequestMapping("/visits")
+@RequiredArgsConstructor
+public class VisitController {
+    private final VisitService visitService;
+    private final VisitMapper visitMapper;
+
+    @PostMapping("/create")
```

Comment:
ja wiem ze to create bo metoda jest POST

Reply:
ok